generators/client/templates/angular/src/main/webapp/app/shared/social/_social.component.ts

```diff
@@ -34,4 +34,8 @@ export class <%=jhiPrefixCapitalized%>SocialComponent implements OnInit {
         this.csrf = this.csrfService.getCSRF();
     }
 
+    submit()
+    {
+        $('#form' + this.label).submit();
```

Comment:
Instead of this frightening jqueryism, please add ngNoForm to the form, that will fix the issue - the original problem is that, as Angular2 founds the 'form' tag, it captures it, and adds it's custom javascript magic. The ngNoForm attribute would prevent this.